assets/prometheus-k8s/rules.yaml

```diff
@@ -511,8 +511,8 @@ spec:
       annotations:
         message: Cluster Monitoring Operator is experiencing {{ printf "%0.0f" $value
           }}% errors.
-      expr: sum(rate(cluster_monitoring_operator_reconcile_errors_total[5m])) * 100
-        / sum(rate(cluster_monitoring_operator_reconcile_attempts_total[5m])) > 10
+      expr: sum(rate(cluster_monitoring_operator_reconcile_errors_total[15m])) * 100
+        / sum(rate(cluster_monitoring_operator_reconcile_attempts_total[15m]))
```

Comment:
@brancz don't we need to compare this value to something? Or are we simply testing that it is non-zero, in which case we don't need to turn it into a percentage, no?

Reply:
copy and paste mistake, you're right

Reply:
Good catch! Fixed that.

Reply:
:+1: